src/AnnotationProcessor/PropertyInformation.php

```diff
@@ -764,14 +764,14 @@ public function getPrecision(): int
      *
      * @return PropertyInformation
      */
-    public function setPrecision(int $precision): self
+    public function setPrecision(?int $precision): self
     {
         // Check range.
         if ($precision < 0 || $precision > 65) {
```

Comment:
`$precision` can be `null` here, so maybe we need to insert the null-default before the IF. Same for the scale?

I think this works is because the `<` and `>` are not strict...